audformat/core/database.py

```diff
@@ -558,6 +607,22 @@ def join_field(
                 else:
                     self[table_id] = table.copy()
 
+        # copy media files
+
+        if copy_media and self.root is not None:
```

Comment:
Maybe we should raise an error if `copy_media=True` but `self.root==None`?

Reply:
You are right, that seems to be the safer option. I changed it accordingly.